core/src/restart_overlay.cpp

```diff
@@ -0,0 +1,46 @@
+#include "qabstractbutton.h"
+
+#include <QCoreApplication>
+#include <QDesktopWidget>
+#include <QFile>
+#include <QMessageBox>
+#include <QPushButton>
+#include <QStyle>
+#include <QTextBrowser>
+
+#include <restart_overlay.h>
+#include <pluginbase/preferences.h>
+#include <stylehelper.h>
+#include <ui_licensedialogoverlay.h>
+#include <application_restarter.h>
+using namespace scopy;
+
+RestartOverlay::RestartOverlay(QWidget *parent)
+	: QDialog(parent)
+	, parent(parent)
+{
+	ui = new Ui::LicenseDialogOverlay();
+
+	ui->setupUi(this);
+	this->setWindowTitle("");
+	ui->description->setText("Please update your graphics drivers or enable hardware acceleration.\n"
```

Comment:
Change text to:

Scopy uses OpenGL for high performance plot rendering. Valid OpenGL context (>v2.0.0) not detected. Restarting will set Scopy rendering mode to software. This option can be changed from the Preferences menu.

Please visit [wiki.analog.com/(](https://wiki.analog.com/university/tools/m2k/scopy-troubleshooting#scopy_white_screen_on_windows) for troubleshooting.